### PR TITLE
[RW-6935][risk=low]remove dua type enum usage

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Review by jmthibault79 — COMMENTED** [review]
LG but please revert the DB change.  Our release rollback safety policy is to remove everything that refers to a DB column first, and only remove the actual DB column after a release cycle has passed.

File: api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java [hunk 1]
```diff
@@ -170,7 +168,6 @@ public long getQueryBatchSize() {
         (rs, unused) ->
             new ReportingInstitution()
                 .displayName(rs.getString("display_name"))
-                .duaTypeEnum(institutionDUATypeFromStorage(rs.getShort("dua_type_enum")))
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java:173]: Tested locally - this reporting field is set to NULL, as expected.

Please also update the column description in workbench-terraform-modules to indicate that this is deprecated.

**Review by jmthibault79 — APPROVED** [review]

File: api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java (hunk 1 above)
Comment (yonghaoy) [reply api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java:173]: It has been there for a while https://github.com/all-of-us/workbench-terraform-modules/blob/main/modules/workbench/modules/reporting/assets/schemas/institution.json#L16

Comment (jmthibault79) [reply api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java:173]: ah thanks!  I was looking at my outdated local copy

**Pull request merged**